apps/web/src/app/api/version/route.spec.ts

```
@@ -1,6 +1,7 @@
 /**
  * @jest-environment node
  */
+'use strict';
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)